SportTrade Interview Assessment

keith-dougherty opened the pull request:
- Implemented List View Controller using a UICollectionView with a compositional layout to allow for future customization

- Collection View uses new (iOS 14) reusable cell registration logic as well as DiffableDataSource to manage updated the view as data changes

- Implemented Detail View Controller to display the selected position's historical data
    **NOTE:** A CocoaPod was added to allow the user to see the changes in price and quantity over time with a line graph     
                      chart

- Implemented a custom Zoom Transition to use when pushing the detail View on/off the navigation stack

- Created a UIManager to handle the creation of reusable UILabel configured to support Dynamic Type

- Organized Project folder structure into meaningful groups

**NOTE:**
 - Changes where made to the stubbed data feed within the listResponse object in order to visualize multiple "stories" within the collection view and to facilitate testing of the search functionality.

keith-dougherty committed:
Implemented List View Controller using a UICollectionView with a compositional layout to allow for future customization
  - Collection View uses new (iOS 14) reusable cell registration logic as well as DiffableDataSource to manage updated the view as data changes

Implemented Detail View Controller to display the selected position's historical data
  - A CocoaPod was added to allow the user to see the changes in price and quantity over time

Implemented a custom Zoom Transition to use when pushing the detail View on/off the navigation stack
Created a UIManager to handle the creation of reusable UILabel configured to support Dynamic Type
Organized Project folder structure into meaningful groups

keith-dougherty committed:
Added additional Comments

willtsporttrade reviewed (changes requested):
When I attempt to build the workspace, I get the error:

> Build input file cannot be found: '/Users/williamtowe/Projects/UIFont+Extensions.swift'

It looks like the path to this file is relative and wasn't included while reorganizing the workspace structure. Could you take a look?

keith-dougherty committed:
Resolved Build Issue

- Resolved missing file issue
- Renamed UITextField Extension to remove extra +
